src/minimega/bridge.go

```diff
@@ -25,10 +25,7 @@ import (
 const (
 	DisconnectedVLAN = -1
 	TrunkVLAN        = -2
-)
-
-const (
-	TAP_REAP_RATE = time.Second
+	TAP_REAP_RATE    = time.Second
```

Comment:
This is fine but I tend to split groups of constants into their logical groupings. The stdlib does this too (groups of things that are IOTA'd, etc.)